src/librustdoc/clean/utils.rs

```diff
@@ -24,17 +24,8 @@ use std::mem;
 mod tests;
 
 crate fn krate(cx: &mut DocContext<'_>) -> Crate {
-    use crate::visit_lib::LibEmbargoVisitor;
-
     let module = crate::visit_ast::RustdocVisitor::new(cx).visit();
 
-    let mut externs = Vec::new();
-    for &cnum in cx.tcx.crates(()) {
-        externs.push(ExternalCrate { crate_num: cnum });
-        // Analyze doc-reachability for extern items
-        LibEmbargoVisitor::new(cx).visit_lib(cnum);
```

Comment:
Are you sure changing the time at which this pass is run has no effect? I'd believe anything of rustdoc 😅

Reply:
Yep, it definitely has an effect! As you said, it's already fixed.